telemetry/telemetry_record.py

```diff
@@ -96,9 +156,356 @@ def failed_url(url, output):
     failed_set[url] = output
     pickle.dump(failed_set, open('failed_urls', 'a'))
 
+def copy_wpr_to_benchmark():
+    """Moves wpr and .pc.wpr (all) files from local data/wpr_source to chromium
+
+    located: /src/tools/perf/page_sets/data/
+    """
+    local_path = os.path.join(os.getcwd(), 'data/wpr_source/*')
+    remote_data_path = os.path.join(CHROMIUM_SRC, 'tools/perf/page_sets/data/')
+    p = Popen('cp {0} {1}'.format(local_path, remote_data_path), shell=True)
```

Comment:
shutil is nicer than Popen for this (e.g., don't need to clean up the process)

https://docs.python.org/2/library/shutil.html